CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+# Contributing
+
+Shamelessly taken from the [coc.nvim](https://github.com/neoclide/coc.nvim) project with sincere thanks.
+
+## How do I... <a name="toc"></a>
+
+- [Use This Guide](#introduction)?
+- Make Something? 🤓👩🏽‍💻📜🍳
+  - [Project Setup](#project-setup)
+  - [Contribute Documentation](#contribute-documentation)
+  - [Contribute Code](#contribute-code)
+- Manage Something ✅🙆🏼💃👔
+  - [Provide Support on Issues](#provide-support-on-issues)
+  - [Review Pull Requests](#review-pull-requests)
+  - [Join the Project Team](#join-the-project-team)
+
+## Introduction
+
+Thank you so much for your interest in contributing!. All types of contributions are encouraged and valued. See the [table of contents](#toc) for different ways to help and details about how this project handles them!📝
+
+The [Project Team](#join-the-project-team) looks forward to your contributions. 🙌🏾✨
+
+## Project Setup
+
+So you wanna contribute some code! That's great! This project uses GitHub Pull Requests to manage contributions, so [read up on how to fork a GitHub project and file a PR](https://guides.github.com/activities/forking) if you've never done it before.
+
+If this seems like a lot or you aren't able to do all this setup, you might also be able to [edit the files directly](https://help.github.com/articles/editing-files-in-another-user-s-repository/) without having to do any of this setup. Yes, [even code](#contribute-code).
+
+### Run Project Locally
+
+If you want to go the usual route and run the project locally, though:
+
+- [Install PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell)
+- [Install Node.js](https://nodejs.org/en/download/)
+- [Install npm](https://npmjs.com)
+- [Fork the project](https://guides.github.com/activities/forking/#fork)
+
+Then in your (PowerShell) terminal:
+
+- Add coc-powershell to your vim's rtp by `set runtimepath^=/path/to/coc-powershell`
+- `cd path/to/your/coc-powershell`
+- `./build.ps1`
+
```

Comment:
"then, start vim/neovim. the local instance of `coc-powershell` will be picked up automatically."
"you can check whether it's loaded by running the command `:CocList extensions` -- a local plugin will be listed with an `[RTP]` tag"
"note, if you have installed `coc-powershell` (managed by `coc.nvim`), you may need to uninstall that first"

Reply:
@corbob you say it better :)
btw, off topic: when writing papers the reviewers often criticize our work for "choppy writing", "hard to understand" etc. etc. -- and I think I still have a long way to go, learn from you guys, and try to understand the subtle differences between me and a native speaker.

Reply:
You do a great job @yatli 😊

Reply:
I agree with Tyler. I simply took what you gave me and spun it into a step by step list of what to do.

It is very much an iterative process. I don't know how many times I've gone back to something I've written and feel that it's really choppy (as in doesn't seem to flow smoothly from topic to topic).